index.html

```diff
@@ -1,15 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
     <title>Robert Kanyoro</title>
-</head>
-<body>
-    <header class="top_Section">
-
-    </header>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <main class="top_Section">
+      <section class="header">
+        <div class="app_bar">
+          <p>9:41</p>
+          <ul class="icons_phone">
+            <li><img src="icons/Cellular Connection.png" alt="Network Cell"></li>
+            <li><img src="icons/Wifi.png" alt="wifi"></li>
+            <li><img src="icons/Battery.png" alt="Battery"></li>
+          </ul>
+        </div>        
+        <nav class="menu_icon">
+          <ul>
+            <li>My Logo</li>
+            <li><img src="icons/Union.png" height="12px" width="18px" alt="Menu"></li>
+          </ul>
+        </nav>
+      </section>
+      <section class="headline">
+        <article class="intro">
+          <h1 id="main_title">Hey There.<br/>I'm Robert.</h1>
+          <h2 class="subtitle">I'm a Software Developer</h2>
+          <p class="desc">I can help you build a product, feature or website. 
+            Look through some of my work and experience! if you like
+            what you see and have a project you need coded, don't hesitate
+            to contact me. 
+          </p>
+          <ul>
+            <li><img src="icons/Github.png" alt="Github Account"></li>
+            <li><img src="icons/Linkedin.png" alt="Linkedin Account"></li>
+            <li><img src="icons/Twitter.png" alt="Github Account"></li>
+          </ul>
```

Comment:
- [ ] Please I highly recommend adding link `a` tag on each `li` for HTML semantics correctly implemented. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Please [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information. 😄 
- [ ] I kindly suggest adding Medium and AngelList icons to the `li` to implement the figma design. 😄